### PR TITLE
New version: Elgato.EpocCam version 3.3.1

### PR DESCRIPTION
**Pull request by Trenly**
﻿- [X] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [X] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [X] Have you tested your manifest locally with `winget install --manifest <path>`?
- [X] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/34715)

### CODE REVIEW

**Review by OfficialEsco — DISMISSED** [review]

**Comment by Trenly** [comment]
Seems to work fine on a sandbox for me; @jamespik or @palenshus  - Would it be possible to get a manual review here?
```raw
--> Installing WinGet
Tip: you can type 'Update-EnvironmentVariables' to update your environment variables, such as after installing a new software.

--> Configuring Winget
Admin setting enabled.

--> Installing the Manifest 3.3.1

Found EpocCam drivers [Elgato.EpocCam] Version 3.3.1
This application is licensed to you by its owner.
Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
Downloading https://edge.elgato.com/egc/windows/epoccam/EpocCam_Installer_3_3_1.exe
  ██████████████████████████████  4.18 MB / 4.18 MB
Successfully verified installer hash
Starting package install...
Successfully installed

--> Refreshing environment variables

--> Comparing ARP Entries

DisplayName           DisplayVersion Publisher  ProductCode
-----------           -------------- ---------  -----------
Bonjour               3.1.0.1        Apple Inc. {56DDDFB8-7F79-4480-89D5-25E1F52AB28F}
EpocCam drivers 3.3.1 3.3.1          Elgato     EpocCam drivers
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
This is what I see on an x86 (Windows 10) VM.

`Architecture` is set to `x86` which means Azure Pipelines boots up an x86 VM:
image

File: manifests/e/Elgato/EpocCam/3.3.1/Elgato.EpocCam.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created with YamlCreate.ps1 v2.0.0 $debug=QUSU.5-1-19041-1320
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Elgato.EpocCam
+PackageVersion: 3.3.1
+MinimumOSVersion: 10.0.0.0
+InstallerType: nullsoft
+Installers:
+- Architecture: x86
```
Comment (ItzLevvie) [inline manifests/e/Elgato/EpocCam/3.3.1/Elgato.EpocCam.installer.yaml:9]:
```suggestion
- Architecture: x64
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Comment by Trenly** [comment]
> This is what I see on an x86 (Windows 10) VM.
> 
> `Architecture` is set to `x86` which means Azure Pipelines boots up an x86 VM:

Ahhhhh, that makes a lot more sense

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**